frontend/lib/map/map/screen_parent_resizer.dart

```diff
@@ -0,0 +1,98 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+
+/// A widget which resizes the child widget twice when the orientation changes.
+///
+/// * Firstly, the child put in a container with the following constraints:
+///   `width = height = max(screenWidth, screenHeight)`. This creates a square.
+/// * Secondly, the child is resized to
+///   `width = parentWidth, height = parentHeight. This sizes the child according to the parent constraints. This
+///   resize only happens if the child is already initialized. If it is not yet initialized then the resize is delayed.
```

Comment:
I think this is outdated now? At least the second line...

Reply:
Yes, thanks!